src/Compilers/Core/Portable/Collections/ImmutableArrayExtensions.cs

```diff
@@ -777,5 +777,50 @@ internal static Location FirstOrNone(this ImmutableArray<Location> items)
 
         internal static int IndexOf<T>(this ImmutableArray<T> array, T item, IEqualityComparer<T> comparer)
             => array.IndexOf(item, startIndex: 0, comparer);
+
+        internal static bool IsSorted<T>(this ImmutableArray<T> array, IComparer<T> comparer)
```

Comment:
It looks like the added lines in this file were moved from elsewhere. In the future, consider leaving a review comment indicating this.